gatsby-config.js

```diff
@@ -26,6 +26,13 @@ module.exports = {
           contentPadding: '2rem',
           maxWidth: '1700px',
         },
+        // This option is set to disallow to prevent crawling of the site during preview
+        // mode
+        robots: {
+          host: 'https://www.docs-preview.newrelic.com',
+          sitemap: 'https://www.docs-preview.newrelic.com/sitemap.xml',
```

Comment:
```suggestion
          host: 'https://docs-preview.newrelic.com',
          sitemap: 'https://docs-preview.newrelic.com/sitemap.xml',
```

Adding back the changes to remove the `www`.